cloudevents/extensions/data-classification.md

```diff
@@ -0,0 +1,89 @@
+# Data Classification Extension
+
+CloudEvents might contain payload which is subjected to data protection
```

Comment:
s/payload/payloads/  & s/is/are/
or
s/payload/a payload/
but I prefer the formet